packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -42,15 +42,19 @@
     user-select: none;
     width: var(--terra-consumer-nav-width, 320px);
     z-index: 2;
-
     @media screen and (max-width: $terra-tiny-breakpoint) {
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
+      height: calc(100% - 60px) !important;
       overflow: auto;
       padding-top: var(--terra-consumer--mobile-nav-padding-top, 0);
       position: absolute;
+      /*stylelint-disable function-url-quotes*/
+      @-moz-document url-prefix() {
+        min-height: max-content;
```

Comment:
are you sure this is needed? Also according to MDN that property value isn't supported in firefox. https://developer.mozilla.org/en-US/docs/Web/CSS/min-height